code/datums/movement_modifier/movement_modifiers.dm

```diff
@@ -79,6 +80,7 @@
 // robot modifiers
 /datum/movement_modifier/robot_base
 	health_deficiency_adjustment = -INFINITY
+	mob_pull_multiplier = 0.2
```

Comment:
How does this affect the AI's bot's ability to shove people around?

Reply:
Should only affect specifically cyborgs *pulling* mobs